tests/rust/re_integration_test/tests/check_focus_test.rs

```diff
@@ -0,0 +1,125 @@
+use re_integration_test::HarnessExt as _;
```

Comment:
I think it would be nice to keep a high-level summary of what the test does at the top of this file. Basically what was previously the `README` in the Python file that this test automates. Otherwise it's a bit hard to understand the context / goal 🤔 

```suggestion
//! This test logs a few boxes and performs the following focus checks:
//!
//! - Double-click on a box in the first view
//!   - check ONLY the corresponding view expands and scrolls
//!   - check the streams view expands and scrolls
//! - Double-click on the leaf "boxes3d" entity in the streams view, check both views expand (manual scrolling might be needed).

use re_integration_test::HarnessExt as _;
```